server/httpsrv/httpsrv.go

```diff
@@ -51,6 +52,7 @@ const (
 
 var (
 	EmptyResponse = map[string]interface{}{}
+	JsonTest, JsonTestCompileErr = regexp.Compile("^application/(?:.*\\+)?json$")
```

Comment:
A couple of things here:
1. The variable is not supposed to be used outside of package and therefore should start with lowercase, lets call it `jsonContentTypePattern`;
2. An error can be returned during initialization that is not accounted for, therefore initialization should be moved to the `init` function. where the error code should be checked and panic originated in case of an error.

Reply:
Pushed a commit that I think meets both.

Is there a Dockerfile that runs the tests? I'm entirely dependent on Docker for Go development.